library/artwork.ts

```diff
@@ -0,0 +1,93 @@
+import Jimp from "jimp";
+import fs from "fs";
+import { AWS_S3_IMAGE_PREFIX } from "../library/constants";
+import s3Client from "../library/s3Client";
+import log from "loglevel";
+import { invalidateCdn } from "./cloudfrontClient";
+
+const localConvertPath = process.env.LOCAL_CONVERT_PATH;
+const cdnDomain = `${process.env.AWS_CDN_DOMAIN}`;
+
+type CONTENT_TYPE = "artist" | "album" | "podcast";
+
+const TYPE_SETTINGS = {
+  artist: {
+    width: 1875,
+    height: Jimp.AUTO,
+    quality: 60,
+  },
+  album: {
+    width: 500,
+    height: 500,
```

Comment:
We should increase our resolution.
```suggestion
    width: 3000,
    height: 3000,
```